docs/src/man/fixednetworkoptim.md

```diff
@@ -152,14 +152,26 @@ Now imagine that our outgroup is taxon A.
   But we could root the best network on the major parent edge to A, edge 10
   (rooted network on the left below).
 - For the second best network in our list, there are 2 ways to root it
-  with A: on the external edge 8 to A (top right), or on its parent edge 10
-  These 2 options give quite different rooted versions
-  of the network, one of which requires the existence of an unsampled taxon,
-  sister to BOECD, that would have contributed to introgression into
-  an ancestor of E. The second rooted version says that an ancestor of
-  (or sister to) A contributed to the introgression into the ancestor of E.
-  A is an outgroup in both cases, but the second case is more parsimonious,
-  in the sense that it does not require the existence of an unsampled taxon.
+  with A.
+  These 2 options give quite different rooted versions of the network: 
+  1. On the external edge 8 to A (top right).   
```

Comment:
@cecileane thoughts on this wording? I think the main difference is that in **2.**  it more emphasizes how the network could be consistent with either having an unsampled taxa or not, unlike  **1.**